javascripts/background_scripts/modal_button.js

```diff
@@ -14,6 +14,9 @@
  *
  */
 
+
+  /*global chrome:false, ls:true, readingProcess:false, */
```

Comment:
I understand that ls:true puts it into the global namespace, but do the other two directives do?